reference/create-index-using-pgroonga.md

```diff
@@ -425,6 +425,11 @@ SELECT * FROM memos WHERE content &@~ 'o123455';
 
 Note that you need to run `REINDEX INDEX pgroonga_memos_index` after you change `normalizations` table. Because normalization results are changed after `normalizations` table is changed.
 
+**Special Case: Using with `pgroonga_highlight_html`**
+
+When you need using `pgroonga_highlight_html` function with this `NormalizerTable`, you need to specify not only `TokenNgram` with `"report_source_location", true"` option but also both `Normalizer` and `NormalizerTable` with `"report_source_offset", true"` option for each.
+
+Please reference [pgroonga_highlight_html](./functions/pgroonga-highlight-html.html) for details.
```

Comment:
Could you use `[label][id]` style link instead of direct link style? `[label][id]` style link makes translation more easier.

```suggestion
Please reference [`pgroonga_highlight_html` function][highlight-html] for details.
```

We also need to add `[highlight-html]:functions/pgroonga-highlight-html.html` to footer like other files.

Reply:
Sorry about that.
I'm not familiar with `[label][id]` style links. I should be careful.